Formula/gcc.rb

```diff
@@ -172,19 +174,7 @@ def install
     # Even when suffixes are appended, the info pages conflict when
     # install-info is run. TODO fix this.
     info.rmtree
-
-    # Rename java properties
-    if build.with?("java") || build.with?("all-languages")
-      config_files = [
-        "#{lib}/gcc/#{version_suffix}/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/security/classpath.security",
-        "#{lib}/gcc/#{version_suffix}/i386/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/i386/security/classpath.security",
-      ]
-      config_files.each do |file|
-        add_suffix file, version_suffix if File.exist? file
-      end
-    end
+    # Since GCC 4.9 java properties are properly sandboxed.
```

Comment:
Still looks like it was added by you 😛 in https://github.com/Homebrew/homebrew-core/commit/59aeccfbab2bb958ed5435760733411b9bd78955 ... I'm building `--with-java` right now so I can see where GCC tries to install the files in question.